### PR TITLE
KafkaAvroSample

### PR DESCRIPTION
**Pull request by markheger**
demonstrate Avro format with Kafka

### CODE REVIEW

File: samples/KafkaAvroSample/com.ibm.streamsx.kafka.sample/KafkaAvroSample.spl [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+use com.ibm.streamsx.avro::TupleToAvro;
+use com.ibm.streamsx.avro::AvroToJSON;
+
+/*
+ * This sample uses AVRO blob for message.
+ */
+composite KafkaAvroSample
+{
+	type
+		Message = blob message, rstring key;
```
Comment (RolefH) [inline samples/KafkaAvroSample/com.ibm.streamsx.kafka.sample/KafkaAvroSample.spl]: Don't define a message type here. Use `MessageType.BlobMessage` for the schema:
```
use com.ibm.streamsx.kafka::MessageType;
// messages get de-serialized into blob
stream <MessageType.BlobMessage> ReceivedMessages = KafkaConsumer() {
    ...
}
```
The same schema can be used for the stream consumed by the KafkaProducer.

Comment (RolefH) [reply samples/KafkaAvroSample/com.ibm.streamsx.kafka.sample/KafkaAvroSample.spl]: Assuming you are doing the change, I'm going to approve the PR.

**Comment by ddebrunner** [comment]
@markheger We should get into the habit of making main composites always `public`, to set an example.

This will then tie in better with `streamsx` which will sometimes nest a main composite in another composite, which then means it has to be public.

**Comment by markheger** [comment]
yes, I missed it.
Will do the changes

**Pull request merged**